gdb toolchain is broken.

[ioamero] [SunOS /tmp/KallistiOS/utils/dc-chain]$ make platform=aica toolchain_profile=stable gdb -j9
******************************************
Sega Dreamcast Toolchains Maker (dc-chain)
******************************************

+++ Using configuration from Makefile.cfg
Using curl as download tool
GNU/Linux build environment detected
+++ Downloading gdb-.tar.xz...
curl --fail --location  -C - -O 'https://ftpmirror.gnu.org/gnu/gdb/gdb-.tar.xz' 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
make: *** [scripts/download.mk:104: gdb-.tar.xz] Error 22

--- a/utils/dc-chain/profiles/aica/stable.mk
+++ b/utils/dc-chain/profiles/aica/stable.mk
@@ -9,6 +9,7 @@ cpu_configure_args=--with-arch=armv4 --with-mode=arm --disable-multilib
 binutils_ver=2.44
 gcc_ver=8.5.0
 newlib_ver=4.5.0.20241231
+gdb_ver=16.3
 
 # GCC custom dependencies
 # Specify here if you want to use custom GMP, MPFR and MPC libraries when

--- a/utils/dc-chain/profiles/dreamcast/15.0.0-lra.mk
+++ b/utils/dc-chain/profiles/dreamcast/15.0.0-lra.mk
@@ -16,6 +16,7 @@ cpu_configure_args=--with-multilib-list=$(precision_modes) --with-endian=little 
 binutils_ver=2.44
 gcc_ver=15.0.0
 newlib_ver=4.5.0.20241231
+gdb_ver=16.3
 
 # Overide SH toolchain download type
 gcc_download_type=git

